.github/workflows/ci.yml

```diff
@@ -45,22 +45,27 @@ jobs:
           mkdir -p $NXF_SINGULARITY_CACHEDIR
           mkdir -p $NXF_SINGULARITY_LIBRARYDIR
 
-      - name: Download test data
+      - name: Download Tiny (RAPID) test data
         # Download A fungal test data set that is full enough to show some real output.
         run: |
           curl https://tolit.cog.sanger.ac.uk/test-data/resources/treeval/TreeValTinyData.tar.gz | tar xzf -
 
+      - name: Singularity - Run RAPID pipeline with test data
+        # Remember that you can parallelise this by using strategy.matrix
+        run: |
+          nextflow run ${GITHUB_WORKSPACE} -entry RAPID -profile test_github,singularity --outdir ./Sing-Rapid
+
       #- name: Docker - Run RAPID pipeline with test data
       # Remember that you can parallelise this by using strategy.matrix
-      #  run: |
-      #    nextflow run ${GITHUB_WORKSPACE} -entry RAPID -profile test_github,docker --outdir ./results-rapid
+      # run: |
+      #  nextflow run ${GITHUB_WORKSPACE} -entry RAPID -profile test_github,docker --outdir ./Dock-Rapid
 
-      - name: Singularity - Run RAPID pipeline with test data
+      - name: Singularity - Run FULL pipeline with test data
         # Remember that you can parallelise this by using strategy.matrix
         run: |
-          nextflow run ${GITHUB_WORKSPACE} -entry RAPID -profile test_github,singularity --outdir ./results-rapid
+          nextflow run ${GITHUB_WORKSPACE} -profile test_github,singularity --outdir ./Sing-Full
```

Comment:
Does this non-rapid test include the steps for rapid test? if so, the rapid test may be not necessary.

Reply:
agree.

Reply:
Yes, rapid is a subset of full. I'll remove rapid from ci.